service-api/module/Application/config/module.config.php

```diff
@@ -27,124 +27,132 @@
 return [
     'router' => [
         'routes' => [
-            'home' => [
-                'type'    => Literal::class,
+            'api' => [
+                'type' => Segment::class,
                 'options' => [
-                    'route'    => '/',
-                    'defaults' => [
-                        'controller' => Controller\IndexController::class,
-                        'action'     => 'index',
-                    ],
+                    'route' => '/api',
```

Comment:
If you're adding this prefix, all the requests in service-front will need to be updated too